codegen.yml

```diff
@@ -1,5 +1,5 @@
 overwrite: true
-schema: "http://localhost:3000/graphql"
+schema: "env"
```

Comment:
as is, this would break the codegen as `"env"` doesnt point to an actual schema set. I think theres some options on how we can pull this in dynamically or programmatically. For the time being, can you revert this change and just leave the localhost url in there?